lib/cli/interactive-setup/deploy.js

```diff
@@ -91,6 +95,7 @@ module.exports = {
     context.serverless = serverless;
 
     printMessage();
+    progress.clear();
```

Comment:
Since I moved the deploy step above the console steps if I didn't clear the progress here there was an `Updating` progress that would persist through the rest of the prompts. I was assuming this was just not seen before because this was the final step 👇 

<img width="1296" alt="Screenshot 2023-03-09 at 9 14 59 AM" src="https://user-images.githubusercontent.com/1753824/224068518-572469db-270c-4a5f-97c6-33ba273166c7.png">

Reply:
Indeed, internal progress is not cleared as it's assumed that it'll be cleared at end of processing in general script logic. So clearing it here makes sense